### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dianasheptak**
[DEMO LINK](https://dianasheptak.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://dianasheptak.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+        </div>
+        <div class="card__price">
+          <p class="card__name-price">Price:</p>
+          <p class="card__number-price">$2,199</p>
+        </div>
+        <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+        >
```
Comment (alexandra-protyanova) [inline src/index.html:44]:
```suggestion
        <a
          href="#"
          class="card__button"
          data-qa="hover"
        >
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+.body {
+  background-color: var(--body-bc-color);
+  border-radius: 5px;
+  font-weight: 400;
+}
+
+.card {
+  width: 200px;
```
Comment (alexandra-protyanova) [inline src/style.css:33]: do not set external geometry to bem blocks, use mixes instead

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white-color);
+  justify-content: center;
+  align-items: center;
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+}
+
+.stars {
```
Comment (alexandra-protyanova) [inline src/style.css:43]: move each bem block to separate file

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+.stars--5 :nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-bottom: 40px;
```
Comment (alexandra-protyanova) [inline src/style.css:72]: combine these margins

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (dianasheptak) [reply src/style.css]: Now the `html corrected` commit is the fixed one (hope so))

**Pull request closed without merging**